clang/lib/AST/StmtOpenACC.cpp

```diff
@@ -324,30 +324,207 @@ OpenACCAtomicConstruct *OpenACCAtomicConstruct::Create(
   return Inst;
 }
 
-static std::pair<const Expr *, const Expr *> getBinaryOpArgs(const Expr *Op) {
+static std::optional<std::pair<const Expr *, const Expr *>>
+getBinaryAssignOpArgs(const Expr *Op, bool &isCompoundAssign) {
```

Comment:
```suggestion
getBinaryAssignOpArgs(const Expr *Op, bool &IsCompoundAssign) {
```
or do bool flags has some exception for naming?

Reply:
*sigh*, nope, just going back and forth between the two naming forms I screwed it up.  I PROMISE I'll get this... one day.